src/components/header/header.js

```diff
@@ -21,7 +21,8 @@ const Header = () => {
                 {headerLinks.map((link) => {
                     return (
                         <HeaderLink
-                            href={link.id}>
+                            href={link.id}
+                            key={headerLinks.indexOf(link)}>
```

Comment:
Dude I messed up. I didn't realize these were divs. Turns out we only need to add the key property when we use the map function with list elements. My bad.